pkg/report/table.go

```diff
@@ -4,16 +4,22 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"path/filepath"
 	"strings"
+	"sync"
 
 	"github.com/fatih/color"
 	"github.com/olekukonko/tablewriter"
 	"golang.org/x/exp/slices"
 
 	dbTypes "github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy/pkg/log"
 	"github.com/aquasecurity/trivy/pkg/types"
 )
 
+// We have to show a message once about using the '-format json' subcommand to get the full pkgPath
+var showMessageOnce sync.Once
```

Comment:
I think we should put it to `TableWriter` as this package is shared with other writers as well.

Reply:
Done